.github/workflows/ccpp.yml

```diff
@@ -69,13 +69,21 @@ jobs:
         run: pip3 install matplotlib
       - name: Flake8 checks
         run: |
+          cd cpp/
```

Comment:
Could we just do `python3 -m flake8 cpp/`?

Reply:
It depends: do you want `cpp/doc/source/conf.py` to be tested too?

Reply:
I have a vague recollection that the order of the imports in `cpp/doc/source/conf.py` is important.

Reply:
I am not sure if that is a yes or a no to my question 😄 
* if you want `cpp/doc/source/conf.py` to be tested, then we could move to `python3 -m flake8 cpp/`, but then it stands to reason that `python/doc/source/conf.py` should be tested too, and thus we should try to move to `python3 -m flake8 python/` too.
* if you do not want `cpp/doc/source/conf.py` to be tested, then we cannot just do `python3 -m flake8 cpp/`.

Reply:
Answer is do not use `isort` on `cpp/doc/source/conf.py` without testing.